index.js

```diff
@@ -28,20 +31,35 @@ function patchFlag(patch, key, config, cb) {
 }
 
 const fetchFlags = function (config, cb) {
-  const { baseUrl, projectKey, sourceEnvironment, destinationEnvironment, apiToken } = config;
+  const { baseUrl, projectKey, sourceEnvironment, destinationEnvironment, apiToken, tags, flag } = config;
+  let isSingle = flag ? true : false;
+  let url = isSingle
+    ? `${baseUrl}/flags/${projectKey}/${flag}?summary=0&env=${sourceEnvironment}&env=${destinationEnvironment}`
+    : `${baseUrl}/flags/${projectKey}?summary=0&env=${sourceEnvironment}&env=${destinationEnvironment}${tags ? ("&filter=tags:" + tags.join('+')) : '' }`;
```

Comment:
Nested ternaries are quite difficult to read and debug so much so [eslint has a rule disallowing it](https://eslint.org/docs/rules/no-nested-ternary). I recommend we avoid nesting ternaries like this and be more verbose using if/else for example.

Reply:
good call - this was hard to follow